include/uprotocol-cpp/uri/serializer/MicroUriSerializer.h

```diff
@@ -139,7 +139,7 @@ class MicroUriSerializer {
      * @param resource 
      * @return 
      */
-    [[nodiscard]] [[maybe_unused]] auto isMicroForm(const uprotocol::v1::UResource &resource) -> bool {
+    [[nodiscard]] [[maybe_unused]] static auto isMicroForm(const uprotocol::v1::UResource &resource) -> bool {
```

Comment:
Remove all [[nodiscard]] and [[maybe_unusued]].

Reply:
remove  [[nodiscard]] and [[maybe_unusued]] from isMicroForm